#include "team_client_player_replied_handler.h"
#include "thread_local/thread_local_storage.h"
#include "network/message_system.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ClientPlayerTeamServiceRepliedHandler::TeamInfoNotify(entt::entity player,
	const ::TeamInfoS2CRequest* request,
	::TeamInfoS2CResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

